### PR TITLE
unit tests for include exclude

### PR DESCRIPTION
**Pull request by jreddig**

### CODE REVIEW

**Comment by danoswaltCL** [comment]
any reason to hold off on merging this to wait on the terminology changes coming with @VivekFitkariwala 's PR?

**Comment by jreddig** [comment]
I dont think it'll generate any conflicts, but I dont mind waiting until the big one gets merged on Monday!

**Review by danoswaltCL — APPROVED** [review]

**Pull request merged**
